Gems/Atom/Feature/Common/Assets/Passes/OpaqueParent.pass

```diff
@@ -342,21 +342,63 @@
                         }
                     ]
                 },
+                {
+                    "Name": "SkyAtmosphereParentPass",
```

Comment:
Should this not be in its own Gem?

Reply:
It can easily be moved to a gem, but I thought that if it would be eventually replacing the existing physical sky implementation then perhaps it makes sense to put in the same place.